src/pipes/is-in-year-and-quarter-range.pipe.ts

```diff
@@ -24,23 +24,24 @@ export function IsInYearAndQuarterRange(
           const year = date.getFullYear().toString();
           const quarter = Math.floor((date.getMonth() + 3) / 3).toString();
           let currentYearValidation = true;
-          if (value.length > 0)
-          if (value && relatedValue) {
-            if (multiSelect){
-            if (value.some(e => e === year)) {
-              currentYearValidation = relatedValue.every(e => e <= quarter);
+          if (value?.length > 0)
```

Comment:
The change here is value?, the rest is prettier. (This failed a unit test.)

There are cases when value may be optional.